submodule.go

```diff
@@ -190,6 +190,9 @@ func getRootImportpath(importpath string) (string, error) {
 		return "", err
 	}
 
+	cwd, _ := os.Getwd()
```

Comment:
do not ignore error here.

Actually, it is 3th call of os.Getwd() may be create global variable `workdir` and initialize it in `init()` function?